Signum.React.Extensions/Translation/TranslationServer.cs

```diff
@@ -26,29 +27,23 @@ public static void Start(IApplicationBuilder app, ITranslator translator)
 
         public static CultureInfo? GetCultureRequest(ActionContext actionContext)
         {
-            foreach (string lang in actionContext.HttpContext.Request.Headers["accept-languages"])
+            var acceptedLanguages = actionContext.HttpContext.Request.GetTypedHeaders().AcceptLanguage;

```

Comment:
Hi Olmo,

we were facing two issues with the previous implementation: 

1. The old implementation used the lower case key "accept-languages" to access the Headers `NameValueCollection`.  If I recall correct, we were facing an issue where the `NameValueCollection` wasn't using a  `CaseInsensitiveComparer` for the lookup, so that the algorithm could not get the accept-languages header at all. 

2. Also, the old implementation did not consider the quality weight of the accept-languages list. Consider the following example: `Accept-Language: de-DE;q=0.5, en-EN;q=0.8`. The old implementation would prefer `de` even though `en` has a higher q-value, because the accept-languages list might be unordered.

Your suggestion is also valid. If an exact region can't be matched, it might make sense to be fuzzy on the region. That would be a `region fallback`, wouldn't it?

Can we combine both approaches?